packages/ibm-products/src/global/js/package-settings.js

```diff
@@ -25,6 +25,7 @@ const defaults = {
     Datagrid: true,
     EditInPlace: true,
     EmptyState: true,
+    EmptyStateV2: true,
```

Comment:
This should be `false` until the component is fully reviewed and released.
```suggestion
    EmptyStateV2: false,
```